R/xgboost-development.R

```diff
@@ -134,10 +132,9 @@ XGBoostDevelopment <- R6Class("XGBoostDevelopment",
       temp_predictions <- predict(self$fitXGB, newdata = self$xgb_testMatrix, reshape = TRUE)
 
       # Build prediction output
-      private$predictions <- temp_predictions %>% 
-        data.frame() %>%
-        mutate(predicted_label = max.col(.),
-               true_label = private$test_label + 1)
+      private$predictions <- as.data.frame(temp_predictions)
+      private$predictions$predicted_label = max.col(private$predictions)
```

Comment:
Comment would be helpful as to what max.col is doing (as well as the +1 that's happening below).

Super excited for xgboost!